characters.js

```diff
@@ -0,0 +1,185 @@
+//   Individual Characters
+const severian = {
+  _name: [
+    "Severian",
+    "The Autarch",
+    "The Conciliator",
+    "The Lictor of Thrax",
+    "Apu-Punchau",
+  ],
+  get name() {
```

Comment:
I'd DRY these getters up somehow. Could implement a `function` like the others below or define a character class which implements it.

Reply:
I agree that both are options are good -- my gut is to lean towards a class but since I haven't started using classes in javascript I'm gonna say that this will also be another good refactor for me in the future